### ISSUE TITLE
DynamicLightsInitializer is not found.

### ISSUE THREAD

**Issue by ye-yu** [LambdAurora/LambDynamicLights#34]
As per [API documentation](https://github.com/LambdAurora/LambDynamicLights/blob/mc1.16/API.md), I should implement `DynamicLightsInitializer` to register my eneity with dynamic lighting, but I cannot find such interface in your mod. I am using

```
SHA256 Hash: 3fbae89e5a236124ddff1b6dd424b3a0fd6575570a102b9f30b2a6e5b32e1adf 
File name: lambdynamiclights-fabric-1.3.1+1.16.3.jar
```

I wish to use your mod as an optional mod, so it is not compulsory to use it with my mod. I load it in my dev environment by attaching your jar to my project through the IDE. Let me know if I should be using another jar file.

To fill in the rest of the template: 
**Desktop (please complete the following information):**
 - OS: Linux
 - Minecraft: 1.16.3
 - Fabric Loader version: 0.25.0+build.415-1.16
 - Mods: None
 - Version: 1.3.1
 - Branch: mc1.16

**Comment by BoogieMonster1O1** [comment]
It looks like `DynamicLightsInitializer` is supposed to be a part of a newer version that hasn't been released yet. 

You can just call the methods in your client mod initializer. It works for me that way.

**Comment by ye-yu** [comment]
Alright, I won't close this until there is such interface in the next release.

**Comment by LambdAurora** [comment]
Available in [1.3.2](https://github.com/LambdAurora/LambDynamicLights/releases/tag/1.3.2) now ;)
